src/service/migrations/eth-to-keys.go

```diff
@@ -0,0 +1,75 @@
+package migrations
+
+import (
+	"context"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/pkg/log"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/service/errors"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/service/formatters"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/utils"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func (c *controller) NewEthereumToKeysOperation() *framework.PathOperation {
+	return &framework.PathOperation{
+		Callback:    c.ethToKeysHandler(),
+		Summary:     "Migrates Ethereum accounts to the keys namespace",
+		Description: "Migrates Ethereum accounts to the keys namespace by copying the data from the ethereum/accounts namespace",
+		Examples: []framework.RequestExample{
+			{
+				Description: "Migrates the current Ethereum accounts to the keys namespace",
+				Response:    nil,
+			},
+		},
+		Responses: map[int][]framework.Response{
+			204: {},
+			500: {utils.Example500Response()},
+		},
+	}
+}
+
+func (c *controller) NewEthereumToKeysStatusOperation() *framework.PathOperation {
+	return &framework.PathOperation{
+		Callback:    c.ethToKeysStatusHandler(),
+		Summary:     "Checks the status of the migration",
+		Description: "Checks the status of the migration for the given namespace",
+		Examples: []framework.RequestExample{
+			{
+				Description: "Checks the status of the migration",
+				Response:    nil,
+			},
+		},
+		Responses: map[int][]framework.Response{
+			204: {},
+			500: {utils.Example500Response()},
+		},
+	}
+}
+
+func (c *controller) ethToKeysHandler() framework.OperationFunc {
+	return func(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+		namespace := formatters.GetRequestNamespace(req)
+
+		ctx = log.Context(ctx, c.logger)
+		err := c.useCases.EthereumToKeys().WithStorage(req.Storage).Execute(ctx, namespace)
```

Comment:
YOu should prevent users are not calling this EP twice. You could check for any existing status before calling the migration UC